tests/ethereum/test_sha3.py

```diff
@@ -477,5 +479,17 @@ def test_example_concrete_1(self):
         self.assertEqual(m.count_all_states(), 1)  # Only 1 state concretized
 
 
+class EthSha3TestFake(EthSha3TestSymbolicate):
+    def setUp(self):
+        evm_consts = config.get_group("evm")
+        evm_consts.sha3 = evm_consts.sha3.fake
+
+        self.mevm = ManticoreEVM()
+        self.worksp = self.mevm.workspace
+
+    def test_example1(self):
```

Comment:
Overall, looks good, but was there supposed to be something here?

Reply:
This test is skipped intentionally.
-- long response
I'm importing all the same tests for "symbolicate" strategy. Though this one (test_example_1) is like 
```
            function foo(uint input) payable public{
                if (sha3(input) == 0x12341234){
                    emit Log("Found a bug");
                }
            }
``` 
It is supposed not to be solved. And with current fakehash implementation the solver simply find a way to invert it. And that should not be the case. I think. So skipped.
Maybe here is a more elegant way to inherit all the others but this one

Reply:
Okay, so, this is ready-to-merge then?

Reply:
Excellent